docs/pagination.md

```diff
@@ -0,0 +1,814 @@
+# Pagination
+
+Pagination is a way to split a large list of items into multiple pages or subsets. This is useful when you have a large number of items to display, and you don't want to overwhelm the user with a long list.
+
+In discord bots, pagination is often used to display a list of items in a more user-friendly way. Instead of displaying all items at once, you can split them information into multiple pages/enbeds, and allow the user to navigate between them.
```

Comment:
```suggestion
In discord bots, pagination is often used to display a list of items in a more user-friendly way. Instead of displaying all items at once, you can split the information into multiple pages/enbeds, and allow the user to navigate between them.
```